src/app/pages/questions/containers/questions-page.component.ts

```diff
@@ -189,9 +191,12 @@ export class QuestionsPage {
   }
 
   navigateToFinishPage() {
-    this.navCtrl.push(FinishPage, {
+    this.navCtrl.push(FinishPageComponent, {
       endText: this.endText,
       associatedTask: this.associatedTask,
+      answers: this.answerService.answers,
```

Comment:
Now since the finish page does not need answers from answer service, perhaps it would be better to reset the answer service before navigating to finish page so as to avoid any merging answers (this is already handled in ionViewDidLoad but just to be sure)